doc/antora/modules/concepts/pages/processing.adoc

```diff
@@ -1,5 +1,6 @@
-= RADIUS Concepts
+= Processing 
 
+Get this diagram or redo
```

Comment:
you should be able to add the SVG to antora.  You can put them into

`doc/antora/modules/references/assets/images`

that directory doesn't exist, but  you can create it, and add/commit files in the directory.

The other antora modules (ROOT, tutorial) use `assets/images` for their images, so we might as well do the same thing here.